ports/glutin/window.rs

```diff
@@ -345,16 +345,16 @@ impl Window {
                     self.pending_key_event_char.set(Some(ch));
                 }
             }
-            Event::KeyboardInput(element_state, scan_code, Some(virtual_key_code)) => {
```

Comment:
Because before we were just handling the case with Some(virtual_key) but now we handle the case None (which doesn't change much the code so I put both on the same case).